backend/event/mutations/CreateEpisodeEntityLinkMutation.py

```diff
@@ -74,3 +75,13 @@ def mutate(cls, root: None, info: ResolveInfo, data: CreateEpisodeEntityLinkInpu
             return cls(ok=False, errors=errors)
 
         return cls(ok=True, errors=[])
+
+    def add_contribution(
+        episode_entity_link: EpisodeEntity,
+        data: CreateEpisodeEntityLinkInput,
+        info: ResolveInfo,
+    ):
+        if info.context:
+            user = info.context.user
+            episode_entity_link.episode.contributors.add(user)
+            episode_entity_link.entity.contributors.add(user)
```

Comment:
> I don't think I've ever encountered a situation in which info.context was falsy. Have you run into this? As far as I can see, the source code does not suggest that it's something we should have to worry about.

Yes, this happens during unit tests (and was causing them to fail). But yes, as far as I know, "real" requests always have a context.

> I am a bit worried about AnonymousUser, however. It should not be a problem because the forms are hidden behind a login, but technically, I guess we could have AnonymousUser as a contributor? Perhaps we should check whether user.is_authenticated(); what do you think?

Hm, I can't imagine we would ever allow anonymous users to make contributions; when we open up the site, anonymous users should have read-only access. So I think it's fine if we don't cover that case here.

`AnonymousUser` is not a database object, by the way (and it's not a `User` either). So if an anonymous request somehow went through, `contributors.add(user)` would raise an error. This would mean the entire transaction is rolled back, and the user receives an error response. That seems fine to me.

If you wanted to allow anonymous contributions, you would need to adjust the database models to store those somehow. But like I said, I don't think this is going to be relevant.